Automattic/wp-calypso
# Help Center: Revert Help Center support fallback

Related to https://github.com/Automattic/wp-calypso/pull/91630

## Proposed Changes

* Reverts changes in: https://github.com/Automattic/wp-calypso/pull/91630

## Why are these changes being made?


*

## Testing Instructions



* Checkout branch
* Sync to Atomic and simple site
* Verify the Help Center opens correctly (Calypso, WP-Admin, Editor)

## Pre-merge Checklist
